CONTRIBUTING.md

```diff
@@ -1,11 +1,12 @@
 ## Getting Involved
-There are many ways to get involved with Ushahidi, and some of them are great even for first time contributors. If you never contributed to Open Source Software before, or need more guidance doing it, please jump in our [gitter](https://gitter.im/ushahidi/Community) channel with a clear description of what you are trying to do, and someone in there will try to help you.
-These are some ways to get involved:
+There are many ways to get involved with Ushahidi projects, and some of them are great even for first time contributors. If you never contributed to Open Source Software before, or need more guidance doing it, please jump in our [gitter](https://gitter.im/ushahidi/Community) channel with a clear description of what you are trying to do, and someone in there will try to help you.
 
-- **Documentation**: if you find an area of the Ushahidi platform that could use better docs, we would love to hear from you in an issue, and would be seriously excited if you send a [Pull Request](https://github.com/ushahidi/platform/compare). This is a great way to get involved even if you are not technical or just have a passion to make information more available and clear to everyone.
+### You can contribute to Ushahidi projects in these areas:
+
+- **Documentation**: if you find an area of the Ushahidi platform that could use better docs, we would love to hear from you in an issue, and would be seriously excited if you send a [Pull Request](https://github.com/ushahidi/platform/compare). This is a great way to get involved non-technically or if you are passionate about making information more available and clear to everyone.
```

Comment:
hey so this is totally not your fault, but could we change "non-technically" to "without knowing how to code" ? or just "...get involved if you are passionate...."  without mention of technical-ity :D 
Writing is a technical skill, and docs are tough, and I don't want folks to feel like their contribution is less technical or less valued <3